Template/LO/SubProcesses/reweight.f

```diff
@@ -1790,7 +1790,7 @@ subroutine update_scale_coupling(p, wgt)
       include 'genps.inc'
       include 'run.inc'
       include 'nexternal.inc'
-      include 'vector.inc' ! needed by coupl.inc (defines VECSIZE_MEMMAX)
+      include 'vector.inc'
```

Comment:
Why not keeping the comment here (would be more clear why this is included)

Reply:
done in #54